cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1616,12 +1641,6 @@ Slice::Gen::ProxyVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
 
     H << eb << ';';
 
-    C << sp;
-    C << nl << "const char*" << nl << scopedPrx.substr(2) << "::ice_staticId() noexcept";
```

Comment:
Why was this removed?

Reply:
I moved this code after the declaration a few lines above in the same file.